Gemfile

```
@@ -24,6 +24,7 @@ gem 'react-rails', '~> 2.6', '>= 2.6.1'
 # Use Active Model has_secure_password
 # gem 'bcrypt', '~> 3.1.7'
 gem 'inline_svg'
+gem 'cancan' 
```

```suggestion
gem 'cancancan' 
```

CanCan is no longer under mantainance, most of its community has migrated to use CanCanCan

From CanCan Documentation: 
> This project is abandoned, see its successor: CanCanCan


https://github.com/CanCanCommunity/cancancan